### PR TITLE
setup New Relic and k6

### PR DESCRIPTION
**Pull request by dmoygit**
PR Highlights
1) installed and setup New Relic
2) installed and setup k6 and created k6ProxyScript.js

### CODE REVIEW

File: ProxyServer/k6ProxyScript.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function() {
+  http.get(`http://localhost:5001?4913406`);
```
Comment (yerx) [inline ProxyServer/k6ProxyScript.js]: Did you not need a '/' in between '5000' and '?'?

File: ProxyServer/newrelic.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      'request.headers.setCookie*',
+      'request.headers.x*',
+      'response.headers.cookie',
+      'response.headers.authorization',
+      'response.headers.proxyAuthorization',
+      'response.headers.setCookie*',
+      'response.headers.x*'
+    ]
+  }
+}
```
Comment (yerx) [inline ProxyServer/newrelic.js]: Would you want put this file in your .gitignore -- if you were paying for the subscription?

File: ProxyServer/k6ProxyScript.js (hunk 1 above)
Comment (dmoygit) [reply ProxyServer/k6ProxyScript.js]: Nice catch!! that may be why my proxy was faster. I'm going to change and retest

File: ProxyServer/newrelic.js (hunk 2 above)
Comment (dmoygit) [reply ProxyServer/newrelic.js]: good point!

**Pull request merged**